src/app/components/Form/Input/index.tsx

```diff
@@ -38,6 +40,17 @@ export function Input({
     },
     [props.type, onChange],
   );
+  const checkAsset = () => {
+    if (appendElem === 'RBTC' || appendElem === 'rBTC') {
+      console.log('appendElem', appendElem);
+      return <AssetRenderer asset={Asset.RBTC} />;
+    } else if (appendElem === 'USDT') {
+      return <AssetRenderer asset={Asset.USDT} />;
+    } else {
+      return appendElem;
+    }
+  };
+
```

Comment:
This file shouldn't have this change, instead you need to use component like this:

```
<Input {some_props} appendElem={<AssetRenderer asset={Asset.USDT} />} />
```